containers/EmpAddress.ts

```diff
@@ -28,7 +28,7 @@ function useEmpAddress() {
   };
 
   return {
-    empAddress: address,
+    empAddress: address ? ethers.utils.getAddress(address) : null,
```

Comment:
Without this, then hardcoding a non-checksummed address in EMPs.ts will cause dApp to crash.